### ISSUE TITLE
TvHeadend plugin skips channels of type "other" - blank guide, no channels

### ISSUE THREAD

**Issue by gstuartj** [MediaBrowser/Emby.Plugins#81]
**Software versions:**
Ubuntu 15.04
Emby 3.0.5621.4
TvHeadend Plugin 1.0.5622.32968
TvHeadend 3.4.28~geb79aee~trusty

**Tuner:** Kworld UB435-Q on U.S. ATSC frequencies.

**Issue Description:**
In some cases TvHeaded imports TV channels with a service type field of "Other." This breaks the TvHeadend plugin for Emby, as it does not have a case for handling type "Other." For me this results in Emby displaying no channels or program guide, since the plugin skips all available channels on refresh.

Each skipped channel creates a log entry like this:
```
2015-05-29 10:33:19.7211 Error - App: [TVHclient] ChannelDataHelper: unkown service type 'Other'.
2015-05-29 10:33:19.7211 Error - App: [TVHclient] ChannelDataHelper: unable to detect service-type from service list:
HTSMessage:
  <dump>
    channelId : 11
    channelNumber : 110
    channelName : ABC
    eventId : 10120
    nextEventId : 10122
    services : 
HTSMessage:
  <dump>
    name : Kworld USB/521,028 kHz
    type : Other
  </dump>

, 
    tags : 
    method : channelAdd
  </dump>
```
The switch statement where the problem occurs is at line 151 of DataHelper/ChannelDataHelper.cs. I do not know whether services set as type "Other" can always be assumed to be TV channels.

**Comment by gstuartj** [comment]
Perhaps this could be solved for edge cases like me by having an advanced setting toggle in the plugin configuration that allows you to import "Other" as TV channels.

**Comment by gstuartj** [comment]
Fixed in TVHclient plugin release 1.0.5628.29938.

[Forum thread](http://emby.media/community/index.php?/topic/20107-tvheadend-plugin-for-emby-beta-1/page-6#entry213697)
